symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/signal/OboSignalService.java

```diff
@@ -0,0 +1,128 @@
+package com.symphony.bdk.core.service.signal;
+
+import com.symphony.bdk.gen.api.model.BaseSignal;
+import com.symphony.bdk.gen.api.model.ChannelSubscriber;
+import com.symphony.bdk.gen.api.model.ChannelSubscriptionResponse;
+import com.symphony.bdk.gen.api.model.Signal;
+
+import org.apiguardian.api.API;
+
+import java.util.List;
+import java.util.stream.Stream;
+
+/**
+ * Service interface exposing OBO-enabled endpoints to manage signals information.
+ */
+@API(status = API.Status.STABLE)
+public interface OboSignalService {
+
+  /**
+   * Lists signals on behalf of the user.
+   * {@link SignalService#listSignals(Integer, Integer)}
+   *
+   * @param skip  The number of signals to skip.
+   * @param limit Maximum number of signals to return. Default is 50, maximum value is 500.
+   * @return List of signals that the user has created and public signals to which they have subscribed.
+   * @see <a href="https://developers.symphony.com/restapi/reference#list-signals">List Signals</a>
+   */
+  List<Signal> listSignals(Integer skip, Integer limit);
```

Comment:
the method names are not very consistent:
- getSignal
- listSignals
- subscribers